spec/features/managing_step_by_step_pages_spec.rb

```diff
@@ -123,6 +123,15 @@
     then_I_see_a_page_reverted_success_notice
   end
 
+  scenario "User publishes changes to a live step by step page" do
+    given_there_is_a_published_step_by_step_page_with_unpublished_changes
+    and_I_visit_the_publish_or_delete_page
+    and_I_click_button "Publish changes"
```

Comment:
There is another method doing something similar: https://github.com/alphagov/collections-publisher/blob/f8401b53dd3f48766d63dc4b9f050a856d190fa7/spec/features/managing_step_by_step_pages_spec.rb#L373, perhaps they could be consolidated?